src/main/java/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> storeActivitiesFromFile;
+        try {
+            storeActivitiesFromFile = Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't reaad data from file", e);
+        }
+        return storeActivitiesFromFile;
+    }
```

Comment:
It is not necessary to create additional variables if you do not perform several operations on them
```suggestion
    public List<String> readFromFile(String filePath) {
        try {
            return Files.readAllLines(Paths.get(filePath));
        } catch (IOException e) {
            throw new RuntimeException("Can't reaad data from file: " filePath, e);
        }
    }
```

Reply:
done, thanks